eos-core-depends

```diff
@@ -58,3 +58,94 @@ unoconv
 unrar
 whiptail
 xdg-app
+
+
+# moved here from os-depends due to installer
+chromium-browser
+cups
+eos-social
+eos-speedwagon
+eos-tech-support
```

Comment:
I think you should leave this in it's just a small set of bash scripts that are helpful when there are issues. According to dpkg, it's only 58kB.